lnpbp-0001.md

```diff
@@ -130,46 +130,57 @@ the **commit procedure** runs as follows:
 
 1. Reduce list `P*` to a set of unique public keys `P`, by removing all duplicate 
    public keys from the list.
-3. Compute sum `S` of all unique public keys in set `P`; fail the protocol if
+2. Compute sum `S` of all unique public keys in set `P`; fail the protocol if
    an overflow over elliptic curve generator point order happens during the 
    procedure.
 3. Construct a byte string `lnpbp1_msg`, composed of the original message
    prefixed with a single SHA256 hash of `LNPBP1` string and a single SHA256
    hash of the protocol-specific `tag`:  
    `lnpbp1_msg = SHA256("LNPBP1") || SHA256(tag) || msg`
-4. Compute HMAC-SHA256 of `lnbp1_msg` using the sum of public keys `S`. The 
-   resulting value is named **tweaking factor** `f`:  
-   `f = HMAC-SHA256(lnpbp1_msg, S)`
-5. Make sure that the tweaking factor is less than the order `n` of a generator 
+4. Convert (serialize) the aggregate public key `S` from step 2 into a 
+   byte-encoded pair of values in uncompressed form `S*`, according to the
+   defacto standard format for uncompressed public keys in Bitcoin,
+   which is followed by libraries like 
+   [rust-secp256k1](https://docs.rs/secp256k1/0.20.1/src/secp256k1/key.rs.html#290-306),
+   resulting in a 65 byte array with:
+    - the first byte having the fixed hex value `04` (0x04),
+    - followed by 32 bytes representing the x coordinate in big-endian order,
+    - followed by 32 bytes representing the y coordinate in big-endian order.
+5. Compute HMAC-SHA256 of `lnbp1_msg` using the sum of public keys in the 
+   serialized format `S*` from step 4. The resulting value is named 
+   the **tweaking factor** `f`:  
+   `f = HMAC-SHA256(lnpbp1_msg, S*)`
```

Comment:
```suggestion
5. Compute HMAC-SHA256 of `lnbp1_msg` using the sum of public keys `S*` from step 4 serialized as 64-byte array, with first 32 bytes occupied by x-coordinate and the last 32 bytes by y-coordinate, both in big-endian order. The resulting value is named 
   the **tweaking factor** `f`:  
   `f = HMAC-SHA256(lnpbp1_msg, S*)`
```

We can do a reference to the rationale section after serialization description and add section there specifying you why we use 64 bytes instead of 65 and why we still follow secp256k1 byte sequence in the remaining 64 bytes (you have that description already)